src/parcsr_mv/_hypre_parcsr_mv.h

```diff
@@ -75,6 +86,11 @@ typedef struct _hypre_ParCSRCommPkg
    HYPRE_Int                         num_recvs;
    HYPRE_Int                        *recv_procs;
    HYPRE_Int                        *recv_vec_starts;
+   HYPRE_Int                         use_neighbor;
+#ifdef HYPRE_USING_NODE_AWARE_MPI
+   long                             *global_send_indices;
```

Comment:
Maybe it's better to use `hypre_uint64`, see https://github.com/hypre-space/hypre/blob/36ab29ba0977294a741987c4f921fe02840dff59/src/utilities/general.h#L33 or `hypre_longint` here